recipes/capytaine/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "1.0" %}
+
+package:
+  name: capytaine
+  version: {{ version }}
+
+source:
+  url: https://github.com/mancellin/capytaine/archive/v{{version}}.tar.gz
+  sha256: 072de801b50b4052277b5eb3d634734a11079b69789e5fb3bccb7363d199f637
+
+build:
+  skip: True  # [py<36]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+  host:
+    - python >= 3.6
+    - pip
+    - setuptools
+    - numpy
+    - llvm-openmp  # [osx]
+  run:
+    - python >= 3.6
+    - attrs
+    - {{ pin_compatible('numpy') }}
+    - pandas
+    - matplotlib
+    - scipy
+    - vtk
+    - xarray
+    - llvm-openmp  # [osx]
+
+test:
+  imports:
+    - capytaine
+  requires:
+    - pytest
```

Comment:
This doesn't seem to be required since the only test that seems to be running is `import capytaine`. Were you intending to add a `run_test.py` or a `commands:` section?

Reply:
It seems I forgot to call pytest...

Reply:
So, it seems that pytest cannot access the tests if they are not in the package. I'll remove the testing for the moment and consider adding the test directory to the package.

Reply:
There are two options: tell `conda build` to copy test files from the recipe (and add them to the recipe): https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-files, or tell `conda build` to copy files from the source: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#source-files

Reply:
Thanks, I'll try the latter.